src/modules/esl-utils/README.md

```diff
@@ -11,73 +11,73 @@ Even if you use the whole utils module, it is still tiny.
 
-  - #### [Promise Utils](./async/promise.ts) - a set of Promise helpers.
+  - ##### [Promise Utils](./async/promise.ts) - a set of Promise helpers.
     Allows converting DOM Events, timeouts, pooling check, etc. to ES6 Promise or create a Deferred object 
     that allows to resolve or reject related Promise.
 
-  - #### [RAF(Rendering callback) Utils](./async/raf.ts) - Request Animation Frame common helpers.
+  - ##### [RAF(Rendering callback) Utils](./async/raf.ts) - Request Animation Frame common helpers.
     Includes RAF Decorator that postpones multiple function calls to a single one before rendering. 
     Also includes `afterNextRender` deferring function.
 
-  - #### [Throttle](./async/throttle.ts) - throttling function decorator.
+  - ##### [Throttle](./async/throttle.ts) - throttling function decorator.
     Throttling is a pattern that allows to limit function call to one execution per timeout.
 
 
-- ### TS Decorators
+- #### TS Decorators
 
-    - #### [@bind](./decorators/bind.ts) - TS decorator to bind method context to current class instance.
+    - ##### [@bind](./decorators/bind.ts) - TS decorator to bind method context to current class instance.
 
-    - #### [@memoize](./decorators/memoize.ts) - TS decorator to make method or get accessor memoized.
+    - ##### [@memoize](./decorators/memoize.ts) - TS decorator to make method or get accessor memoized.
 
-    - #### [@prop](./decorators/prop.ts) - TS decorator to define a field on the prototype level.
+    - ##### [@prop](./decorators/prop.ts) - TS decorator to define a field on the prototype level.
 
-- ### DOM Helpers
+- #### DOM Helpers
   
-    - #### [Keys](src/modules/esl-utils/dom/keys.ts) - keyboard keys constants.
+    - ##### [Keys](./dom/keys.ts) - keyboard keys constants.
   
-    - #### [RTL Utils](./dom/rtl.ts) - Utils to detect RTL and RTL-specific browsers behavior.
+    - ##### [RTL Utils](./dom/rtl.ts) - Utils to detect RTL and RTL-specific browsers behavior.
   
-    - #### [Scripts](./dom/script.ts) - script loading utility to limit and track loading.
+    - ##### [Scripts](./dom/script.ts) - script loading utility to limit and track loading.
   
-    - #### [Scroll](./dom/scroll.ts) - scroll locking methods. 
+    - ##### [Scroll](./dom/scroll.ts) - scroll locking methods. 
       *Note: uses [scroll.less](./dom/scroll.less) styles.*
   
-    - #### [CSSClass](./dom/class.ts) - is a utility to work with CSS classes. Supports JQuery-like enumeration, inversion syntax and locks.
+    - ##### [CSSClass](./dom/class.ts) - is a utility to work with CSS classes. Supports JQuery-like enumeration, inversion syntax and locks.
   
-    - #### [DOM Traversing](./dom/traversing.ts) - a set of utils to find DOM elements or check their relations.
+    - ##### [DOM Traversing](./dom/traversing.ts) - a set of utils to find DOM elements or check their relations.
```

Comment:
Looks like that we missed element, window, and rect helpers... :(

Reply:
The final version will be in a separate PR.
Thanks for the good point!